pages/article/[pid].tsx

```diff
@@ -73,10 +74,10 @@ border-radius: 22px;
 const StyledEmoji2 = styled.div`
 background: red;
 border-radius: 22px;
-padding: 0.2em 0.4em 0.2em 0.4em;
+padding: 0.5em;
 margin :0.5em;
```

Comment:
The emoji should have a hover click effect

Reply:
should there be a change i color on hover ?